src/codegen/llvm/jit_driver.cpp

```diff
@@ -158,8 +158,10 @@ void JITDriver::init(std::string features,
     // with listeners on (e.g. -DLLVM_USE_PERF=ON).
     if (benchmark_info) {
         gdb_event_listener = llvm::JITEventListener::createGDBRegistrationListener();
+#if defined(NMODL_HAVE_JIT_EVENT_LISTENERS)
```

Comment:
It is actually safe to not use #if since if flag for listeners not set in LLVM build, these will be `nullptr`. But it is better not to rely on that, I agree!

Reply:
But, note that if `createPerfJITEventListener()` and `createIntelJITEventListener()` are **used** but we are not linking to libraries or don't have them then we get undefined reference error. This is why added it.

Reply:
I see, that's right!